keras_nlp/utils/text_generation.py

```diff
@@ -228,9 +231,9 @@ def beam_search(
         token_probability_fn: a callable, which takes in input_sequence
             and output the probability distribution of the next token. If
             `from_logits` set to True, it should output the logits of the next
-            token. The input shape would be `[batch_size, length]` and the
-            output should be `[batch_size, vocab_size]`, where batch_size is
-            variable.
+            token. The input shape would be `[batch_size * num_beams, length]`
+            and the output should be `[batch_size * num_beams, vocab_size]`,
+            where batch_size is variable.
```

Comment:
I'm not sure what "where batch_size is variable" is attempting to communicate, let's just remove.